init.sqf

```diff
@@ -93,9 +93,18 @@ f_script_briefing = [] spawn f_fnc_createBriefing;
 
 // ====================================================================================
 
-// F3 - Revive
+// F3 - FA Medical 
 // Credits and documentation: https://github.com/folkarps/F3/wiki
-[] execVM "f\medical\init.sqf";
+
+{
+	_x remoteExec ["f_fnc_famInit",player,player];
+} foreach playableUnits;

```

Comment:
It is in this state because JIP was not working. This does not work for JIP either though, so still looking for JIP safe method. Event handler?

Reply:
something like this then?
`player remoteExec ["f_fnc_famInit",0];` this tells eveyrone that this player exists. so each other machine would then install the necessary handlers for that new player

Reply:
I would like to understand the current code here which reads `_x remoteExec ["f_fnc_famInit",0,player];`.
So each player does a remoteExec on everyone else for every playable unit that exists. Does this mean that it runs n^3 times for n players?
Did my previous suggestion not work?

Reply:
I would like to understand the current code here which reads `_x remoteExec ["f_fnc_famInit",0,player];`.
So each player does a remoteExec on everyone else for every playable unit that exists. Does this mean that it runs n^3 times for n players?
Did my previous suggestion not work?

Reply:
I don't quite understand the latest change which reads `_x remoteExec ["f_fnc_famInit",0,player];`.
If I udnerstand it correctly every player runs f_fnc_famInit on every client for every playable unit.
Does this mean that for n players it effectively does n^3 remoteExec's? And then out of these n^3 invocations we discard n^3-1 of them since the code itself checks if the unit is local?
I think what we want to do here instead is simply `_x call f_fnc_famInit`;